docs/tutorials/06_examples_max_cut_and_tsp.ipynb

```diff
@@ -136,13 +136,396 @@
+      "image/svg+xml": [
```

Comment:
I don't think you need to modify the image. Btw, I turned off the svg format because it causes a lot of line changes. I'm wondering why your commit still includes svg part. On my local mac, the svg part is not generated.

Reply:
I have no idea - I would have to look at jupyter notebook and see if there are any settings in this regard that it creates this when it saves, All I did was to edit the title and save it, nothing more.

Reply:
This might be due to notebook version. I committed this tutorial with notebook 6.3 or 6.4. What is your version?

Reply:
Seems quite a bit older 4.4

Reply:
If you update the notebook version, the diff may get smaller. But, I approve this change because the it's not so important.

Reply:
Yes, it might be a good idea for me to update - as long as I don't end up with all those random ids in the file. I think they may come from nbformat - what version is yours?

Reply:
Mines are as follows.
```
notebook                      6.4.0
nbconvert                     6.1.0
nbformat                      5.1.3
```

Reply:
FYI: I updated to those versions. Loaded it up and saved it and its unchanged from before; made changes to it just in case but its still the same for me in regards the image sections. So I tried...

Reply:
Thank you for trying the latest version.